modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -299,6 +299,9 @@ When you are ready to deploy an application, follow the instructions in xref:dep
 
 The *Ingress* tab allows you to specify whether inbound traffic should be allowed for the application. After you enable inbound traffic, the default behavior is changed to allow for new application deployments. If there are applications deployed to Runtime Fabric before you enable inbound traffic, they do not receive inbound requests until this setting is enabled.
 
+== Upgrade Considerations
+For Runtime Fabric versions 1.5.0 or later, the inbound traffic software version is only upgraded after the Runtime Fabric version is upgraded. The next inbound traffic software deployment (following the Runtime Fabric version upgrade) triggers the new inbound traffic software version to run. Runtime Fabric versions earlier than 1.5.0 will trigger inbound traffic software version updates at deployment times independent of Runtime Fabric version upgrades.
```

Comment:
Suggestions:
 * "Upgrade Considerations" -> "Changes to Upgrades" 
"inbound traffic software" -> "internal load balancer" 
"is only upgraded after the Runtime Fabric version is upgraded" -> "is upgraded during the Runtime Fabric component upgrade process"
Consider removing the last 2 sentences.

Reply:
Updated - thank you!